mem/log.go

```diff
@@ -147,11 +151,11 @@ func (log *memlog) Query(specs ...margaret.QuerySpec) (luigi.Source, error) {
 	return qry, nil
 }
 
-func (log *memlog) Append(v interface{}) (margaret.Seq, error) {
+func (log *memlog) Append(v interface{}) (int64, error) {
 	log.l.Lock()
 	defer log.l.Unlock()
 	if log.closed {
-		return nil, io.ErrClosedPipe // already closed
+		return -2, io.ErrClosedPipe // already closed
```

Comment:
why -2 btw? seen this a couple of times now

Reply:
is this equivalent to `margaret.SeqSublogDeleted` that i've seen elsewhere?

Reply:
and if not: does a constant make sense to introduce? lotta -2 scattered around ^^

Reply:
agreed, there should be a defined constant here.

`read()` and `write()` return negative counts when the call fails, that's the analogy here at least.

butttt `-1` is _empty log_ so `-2` should mean something like _something went wrong_. Good call, i will define it.